benefits/admin.py

```diff
@@ -0,0 +1,25 @@
+import requests
+
+from django.conf import settings
+from loguru import logger
+
+
+def pre_login_user(user, request):
+    logger.debug(f"Running pre-login callback for user: {user.username}")
+    token = request.session.get("google_sso_access_token")
+    if token:
+        headers = {
+            "Authorization": f"Bearer {token}",
+        }
+
+        # Request Google user info to get name and email
+        url = "https://www.googleapis.com/oauth2/v3/userinfo"
+        response = requests.get(url, headers=headers, timeout=settings.REQUESTS_TIMEOUT)
```

Comment:
@thekaveman What should I set the timeout to? I just went with the app default here.

Tests are failing: https://github.com/cal-itp/benefits/actions/runs/7717803049/job/21037685917#step:4:181

Reply:
@machikoyasuda I think that test failure is the same one I was seeing yesterday related to the fix from #1862.

If you rebase on `dev` the tests should pass again? Your usage of the app default looks correct to me.